### PR TITLE
Created a series of linters to perform various validations 

### PR DESCRIPTION
**Pull request by ak-dassana**

### CODE REVIEW

**Comment by ak-dassana** [comment]
@parth-dassana G just reminded me of this but the file config-recording-not-enabled.yaml is not valid according to the resource-hierarchy.yaml mapping. Not sure which file to update so need your input

File: engine/src/main/java/app/dassana/core/api/linter/VendorLinter.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+import java.util.stream.Collectors;
+
+public class VendorLinter extends BaseLinter {
+
+	private Set<String> template = new HashSet<>(), ignore, required;
+	private Gson gson = new Gson();
+
+	public void buildIgnoreList(){
+		String[] ignoreList = new String[]{"config-recording-not-enabled.yaml", "user-has-console-and-access-keys.yaml",
+		"iam-role-not-enabled-for-ec2.yaml", "vpc-route-table-is-overly-permissive.yaml"};
```
Comment (parth-dassana) [inline engine/src/main/java/app/dassana/core/api/linter/VendorLinter.java:17]: This cannot be here for sure. Please delete the files or something of that sort in the future.

File: engine/src/main/java/app/dassana/core/api/linter/VendorLinter.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+
+		String[] requireList = new String[]{"alertId", "canonicalId", "vendorId", "vendorPolicy", "csp",
+						"resourceContainer", "region", "service", "resourceType", "resourceId"};
```
Comment (parth-dassana) [inline engine/src/main/java/app/dassana/core/api/linter/VendorLinter.java:20]: Pretty sure this is handled by `engine/src/main/java/app/dassana/core/workflow/processor/RequestProcessor.java`

**Review by parth-dassana — CHANGES_REQUESTED** [review]
Couple things

**Pull request closed without merging**